libraries/abstractions/pkcs11/utest/iot_pkcs11_mbedtls_utest.c

```diff
@@ -842,14 +927,17 @@ void test_pkcs11_C_CreateObjectECPrivKey( void )
     pvPortMalloc_Stub( pvPkcs11MallocCb );
     PKCS11_PAL_FindObject_IgnoreAndReturn( 1 );
     PKCS11_PAL_GetObjectValue_IgnoreAndReturn( CKR_OK );
-    mbedtls_pk_parse_key_IgnoreAndReturn( 0 );
+    mbedtls_pk_parse_key_IgnoreAndReturn( 1 );
+    vLoggingPrintf_CMockIgnore();
+    mbedtls_strerror_highlevel_IgnoreAndReturn( NULL );
+    mbedtls_strerror_lowlevel_IgnoreAndReturn( NULL );
     PKCS11_PAL_GetObjectValueCleanup_CMockIgnore();
-    pvPortMalloc_Stub( pvPkcs11MallocCb );
+    pvPortMalloc_IgnoreAndReturn( &xKeyContext );
     mbedtls_ecp_keypair_init_CMockIgnore();
     mbedtls_ecp_group_init_CMockIgnore();
     mbedtls_ecp_group_load_IgnoreAndReturn( 0 );
-    mbedtls_mpi_read_binary_IgnoreAndReturn( 0 );
     pvPortMalloc_Stub( pvPkcs11MallocCb );
```

Comment:
here too, 
stubs can only be set at the beginning of the function , no need to redefine them before every function call

Reply:
This one should stay the same as I have different behavior in the malloc stub before this one

Reply:
what do you mean?

Reply:
pvPortMalloc_IgnoreAndReturn( &xKeyContext ); Above this one injects a struct